### PR TITLE
[MPT-22] README update

### PR DESCRIPTION
**Pull request by Hackin7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,91 @@
...
+our code changes
+
+## Getting Started
+
+To get started, make sure NodeJS is installed on your computer.
+You can install it following the instructions [here](https://nodejs.org/en/)
+
+Afterwards, we can download this repo onto our computer, or we can clone
+this repo using the command
+`git clone https://github.com/AdvisorySG/mentorship-page.git`
```
Comment (wei2912) [inline README.md:45]: This should be block code for ease of copying.

Also, please specify the `$` sign infront of the commands (following usual Bash command prompt conventions)

**Review by wei2912 — CHANGES_REQUESTED** [review]
Generally looks accurate and clear to me, though capitalisation in some areas seem to be inconsistent (e.g. "Airtable" vs "AirTable"). Also, try to avoid using `here` and then adding a link; it would be clearer to use the title of the page, which gives more information about what that page provides. See rest of comments in this review.

**Comment by Hackin7** [comment]
I made most of the changes requested. For now I put Node v18 as the version, so have to wait for https://github.com/AdvisorySG/mentorship-page/pull/534 to be pushed first

I realised that it may be important to add is Contributing guidelines, but I'm not too sure how to best do it for now. For now I linked the Ghost Theme CONTRIBUTING.md (which I think is quite good), but I also added in a small section about `prettier`, and that they should run it before submitting the PR. (Especially since there's a check for it)

**Comment by wei2912** [comment]
> For now I linked the Ghost Theme CONTRIBUTING.md (which I think is quite good), but I also added in a small section about `prettier`, and that they should run it before submitting the PR. (Especially since there's a check for it)

The `CONTRIBUTING.md` document is meant to be org-wide but it hasn't been updated in a while (think I wrote it back in 2020...). Feel free to make any changes on the Ghost repository and create a new PR here to add a copy to here.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,104 @@
...
+```bash
+$ npm install # Update and Install the required dependency packages
+$ npm start
+```
+
+## Contributing
+
+This project uses the [GitHub flow](https://guides.github.com/introduction/flow/).
+Contributors should create a new branch for every Pull Request to be submitted.
+Before submitting the Pull Request, make sure to run `prettier` to format the code for readability purposes.
```
[the remark this answers is not in the document]
Comment (Hackin7) [reply README.md]: For now I've added a section about running prettier manually (if commiting to github directly), hopefully it is useful for new developers to the repo who may not know about the hook

**Comment by wei2912** [comment]
Blocked by https://github.com/AdvisorySG/mentorship-page/pull/559

**Comment by wei2912** [comment]
Somehow pushing from my copy to your branch causes all these extra commits to pop up (instead of the branch adopting the actual commits of the `master` branch)... apologies for that

File: README.md [hunk 3]
```diff
@@ -33,33 +134,3 @@ If you aren’t satisfied with the build tool and configuration choices, you can
 Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
```
Comment (wei2912) [inline README.md:136]: Same for "Available Scripts". A lot of these aren't really relevant (they were artifacts from Create React App).

**Review by wei2912 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**